### ISSUE TITLE
OPDS feed does not have categories for common archives such as many Zimit archives

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-js-windows#908]
Approximately 831 uncategorized ZIMs are in the library and are invisible to the current OPDS-based library feed.

We need to provide an All categories entry in the library so users can find these archives, some of which are very valuable.

Although this is a regression compared to previous directory-based library, it is not a code regression per se.
